src/PrinterLog.php

```diff
@@ -121,7 +121,22 @@ public function getMetrics(Printer $printer, $user_filters = []): array
             ] + $filters
         ]);
 
-        return iterator_to_array($iterator);
+        $series = iterator_to_array($iterator);
+
+        // Reduce the data to 25 points
+        $count = count($series);
+        $max_size = 25;
+        if ($count > $max_size) {
+            // Keep one row every X entry using modulo
+            $modulo = round($count / $max_size);
+            $series = array_filter(
+                $series,
+                fn($k) => ($k % $modulo) == 0,
```

Comment:
To be sure that last point will always be display, filtering could be change to this:
```suggestion
                fn($k) => (($k + 1 - $count) % $modulo) == 0,
```

For instance, with 48 entries (last entry index is 47):

using current formula
 - `0 % 2 == 0 => true`
 - `1 % 2 == 0 => false`
 - `2 % 2 == 0 => true`
 - `3 % 2 == 0 => false`
 - ... 
 - `46 % 2 == 0 => true`
 - `47 % 2 == 0 => false`

using proposed formula
 - `( 0 + 1 - 48) % 2 == 0 => false`
 - `( 1 + 1 - 48) % 2 == 0 => true`
 - `( 2 + 1 - 48) % 2 == 0 => false`
 - `( 3 + 1 - 48) % 2 == 0 => true`
 - ... 
 - `(46 + 1 - 48) % 2 == 0 => false`
 - `(47 + 1 - 48) % 2 == 0 => true`